scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -2015,6 +2098,24 @@ then
 
     rm -f "${DATA}/${IST_FILE}"
 
+    # Get keyring component status
+    get_mysqld_path
```

Comment:
We are not checking the return code of `get_mysqld_path` here. Original code exited the script if mysqld was not found.

Reply:
You are right. I will make the script to exit if mysqld binary was not found.

Reply:
Done. Modified the `return` to `exit`.